imx_sdp.c

```diff
@@ -673,117 +673,118 @@ void perform_mem_work(struct sdp_dev *dev, struct mem_work *mem)
 	}
 }
 
-static int write_dcd_table_ivt(struct sdp_dev *dev, struct ivt_header *hdr, unsigned char *file_start, unsigned cnt)
+static int write_dcd(struct sdp_dev *dev,
+	struct ivt_header *hdr, unsigned char *file_start,
+	unsigned cnt)
```

Comment:
This change is aesthetic, not related to the commit log, please provide it as a separate patch.